packages/create-amplify/src/logger.ts

```diff
@@ -24,12 +35,68 @@ export class Logger {
       return;
     }
 
-    const logMessage = `[${
-      LogLevel[level]
-    }] ${new Date().toISOString()}: ${message}`;
+    const logMessage =
+      level === LogLevel.DEBUG
+        ? `[${LogLevel[level]}] ${new Date().toISOString()}: ${message}`
+        : message;
     this.console.log(logMessage);
   }
 
+  /**
+   * Start animating ellipsis at the end of a log message.
+   */
+  async startAnimatingEllipsis(message: string) {
+    if (!this.isTTY()) {
+      await this.log(message, LogLevel.INFO);
+      return;
+    }
+
+    this.animateMessage = message;
+    const frames = ['.', '..', '...'];
+    let frameCount = 0;
```

Comment:
Didn't think of this way to implement it. I think this would look cleaner if we ever decide to expand the length of dots (to 9 or something), this would avoid having a long array of strings in the code and easier to make that kind of change.

Reply:
Done in [fb89cac](https://github.com/aws-amplify/amplify-backend/pull/595/commits/fb89cacbc4748373aa55231303d37aa8d05dccbc)